services/movie.py

```diff
@@ -23,18 +26,21 @@ def get_movie_by_id(movie_id: int) -> Movie:
 
 
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None,
```

Comment:
Using mutable default arguments like `list` can lead to unexpected behavior. It's safer to use `None` as a default value and initialize the list inside the function.